_includes/manuals/1.10/4.2.4_smart_vars.md

```diff
@@ -3,11 +3,11 @@ Smart variables are a tool to provide global parameters (key/value data), normal
 
 Smart variables are associated with a Puppet class, but they result in a global parameter.  They may have multiple possible values, all depending on hierarchical context or various conditions a user can wish to apply.  For example:
 
-![Example](/static/images/screenshots/4.2.4_smartvar.png)
+![Example](/static/images/screenshots/param_classes_new/4.2.4_smartvar.png)
```

Comment:
Ah, there's another directory with the "old" set.  Perhaps call it param_classes_110, then we'll know when it was done.